modules/apiserversource-yaml.adoc

```diff
@@ -137,7 +137,7 @@ $ oc apply -f service.yaml
 
 [source,yaml]
 ----
-apiVersion: eventing.knative.dev/v1alpha1
+apiVersion: eventing.knative.dev/v1
```

Comment:
With v1 Trigger, the spec.broker is a required field. (it no longer defaults to "default" broker, the broker needs to be explicitly defined

Reply:
```
apiVersion: eventing.knative.dev/v1
kind: Trigger
metadata:
  name: event-display-trigger
  namespace: default
spec:
  broker: default
  subscriber:
    ref:
      apiVersion: serving.knative.dev/v1
      kind: Service
      name: event-display
```